lib/tasks/search.rake

```diff
@@ -1,170 +1,36 @@
 require 'task_helpers'
 
 namespace :search do
-  def import_from_cursor(cursor, index, opts)
-    tot = cursor.count
-    cnt = 0
-    t = Time.now
-    index.import cursor, {:method => :paginate, :per_page => opts[:batch_size]} do |documents|
```

Comment:
@max: I know we may not be using reindex for the migration.  At the same time, the code was changed, and I think we should leave it in a state we are happy with.

1. Do we need to bring back the ability to sleep between batches?
2. Do we need to bring back feedback while processing?
3. Do we need to output errors?

I know you have ideas of working off the secondary, but I'm not sure how that compares.  I want to do the minimum acceptable at this time.